### PR TITLE
Add exam pagination.

### PR DESCRIPTION
**Pull request by kasya**
Create get_attempt method. Add exam_attempt serializer. Add tests.

### CODE REVIEW

File: apps/exam/serializers.py [hunk 1]
```diff
@@ -15,3 +15,25 @@ class SubjectSerializer(serializers.ModelSerializer):
   class Meta:
     model = Subject
     fields = ('id', 'name')
+
+
+class ExamAttemptSerializer(serializers.ModelSerializer):
+
+  questions = serializers.Serializer()
```
Comment (arkid15r) [inline apps/exam/serializers.py]: Shouldn't the 'questions' be handled by Meta's fields?

File: apps/exam/serializers.py [hunk 2]
```diff
@@ -15,3 +15,25 @@ class SubjectSerializer(serializers.ModelSerializer):
...
+  time_left_seconds = serializers.SerializerMethodField()
```
Comment (arkid15r) [inline apps/exam/serializers.py]: Sort

File: apps/question/views.py [hunk 3]
```diff
@@ -67,3 +69,24 @@ def post(self, request, **kwargs):
...
 
     return Response(status=status.HTTP_201_CREATED)
+
+
+class QuestionAnswerView(APIAttemptBase):
+
+  def get(self, request, **kwargs):
+    """Returns answered question and picked answers from AttemptAnswer."""
+
+    attempt = self.attempt
```
Comment (arkid15r) [inline apps/question/views.py]: Not that important, the code still would be quite readable without it

File: apps/question/views.py [hunk 4]
```diff
@@ -67,3 +69,24 @@ def post(self, request, **kwargs):
...
+    question = Question.objects.get(id=self.kwargs['question_id'])
```
Comment (arkid15r) [inline apps/question/views.py]: What if question with that id doesn't exist?

File: apps/question/views.py [hunk 5]
```diff
@@ -67,3 +69,24 @@ def post(self, request, **kwargs):
...
+
+    if question not in attempt.questions.all():
+      raise Http404
+
+    answer_attempt = AnswerAttempt.objects.get(attempt=attempt.id,
```
Comment (arkid15r) [inline apps/question/views.py]: What about checking attempt existence and user permissions to access that attempt?

Comment (kasya) [reply apps/question/views.py]: I already check that in parent class dispatch() when I get self.attempt.

**Review by arkid15r — APPROVED** [review]

**Pull request merged**